test/validatorTest.js

```diff
@@ -720,6 +720,24 @@ describe("attestation validation", function() {
 					return assert.isRejected(attResp.validateFlags(), Error, "expected flag was not set: ED");
 				});
 
+				it("throws if UV and UV is set but not UP", function() {
+					attResp.expectations.set("flags", ["UV"]);
+					attResp.authnrData.set("flags", new Set(["UV"]));
+					return assert.isRejected(attResp.validateFlags(), Error, "expected User Presence (UP) flag to be set if User Verification (UV) is set");
+				});
+
+				it("throws if UV but UV is not set", function() {
```

Comment:
Same here - I think the wording can be a bit more concise

Reply:
Same here, maybe I can write: "_throws if UV and UV is not set_".

Do you have any suggestions to improve the test name?

Reply:
"throws if UV is not set"? Does it always HAVE to be set, by the way?

Reply:
No, UV is not mandatory. But if UV is present UP must also be present, as per webauthn specification both for [registration (14th element in the check list)](https://www.w3.org/TR/webauthn-2/#sctn-registering-a-new-credential) and for [authentication (16th element in the check list)](https://www.w3.org/TR/webauthn-2/#sctn-verifying-assertion).